### PR TITLE
Update AutoComplete example to use WriteDisposition.APPEND

### PR DESCRIPTION
**Pull request by swegner**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by swegner** [comment]
R: @bjchambers 
CC: @dhalperi

File: examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java [hunk 1]
```diff
@@ -494,7 +494,7 @@ public static void main(String[] args) throws IOException {
                .to(tableRef)
                .withSchema(FormatForBigquery.getSchema())
                .withCreateDisposition(BigQueryIO.Write.CreateDisposition.CREATE_IF_NEEDED)
-               .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_TRUNCATE));
+               .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_APPEND));
```
Comment (bjchambers) [inline examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java]: Consider leaving this as WRITE_TRUNCATE for streaming:

```
.withWriteDisposition(options.isStreaming()
    ? BigQueryIO.Write.WriteDisposition.WRITE_APPEND
    : BigQueryIO.Write.WriteDisposition.WRITE_TRUNCATE);
```

Also, fix the NullPointerException below...

Comment (swegner) [inline examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java]: I've marked the inputFile option as required, as it is used in both the streaming and batch configurations. This should fix the NPE.

**Comment by swegner** [comment]
Jenkins is green; @bjchambers PTAL

**Comment by swegner** [comment]
BigQueryIO changes split off to: https://github.com/apache/incubator-beam/pull/451

**Comment by bjchambers** [comment]
LGTM

**Comment by dhalperi** [comment]
LGTM

**Pull request closed without merging**